### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by nCastle1**
This PR includes improvements for several UI and quality issues identified during certification.

### CODE REVIEW

File: src/Forms/Shared/Samples/Geometry/GeodesicOperations/GeodesicOperations.xaml.cs [hunk 1]
```diff
@@ -21,82 +21,82 @@ namespace ArcGISRuntime.Samples.GeodesicOperations
         "Geometry",
         "This sample demonstrates how to use the Geometry engine to calculate a geodesic path between two points and measure its distance.",
         "Tap on the map to set the end point of a path from New York City. The geodesic path and geodesic distance will be displayed.")]
-    public partial class GeodesicOperations : ContentPage
+public partial class GeodesicOperations : ContentPage
```
Comment (fragmer) [inline src/Forms/Shared/Samples/Geometry/GeodesicOperations/GeodesicOperations.xaml.cs]: Revert indentation changes on this file.

File: src/Forms/Shared/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs [hunk 2]
```diff
@@ -134,7 +134,7 @@ private void GetSuitableTransformations(SpatialReference inSpatialRef, SpatialRe
             {
                 DatumTransformationListBoxItem item = new DatumTransformationListBoxItem(transform)
                 {
-                    IsDefault = (transform.Name == defaultTransform.Name)
+                    IsDefault = transform.Name == defaultTransform.Name
```
Comment (fragmer) [inline src/Forms/Shared/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs]: Keep parens.

File: src/UWP/ArcGISRuntime.UWP.Viewer/App.xaml.cs [hunk 3]
```diff
@@ -34,11 +34,9 @@ public App()
             TaskScheduler.UnobservedTaskException += TaskScheduler_UnobservedTaskException;
         }
 
-        private async void TaskScheduler_UnobservedTaskException(object sender, UnobservedTaskExceptionEventArgs e)
+        private void TaskScheduler_UnobservedTaskException(object sender, UnobservedTaskExceptionEventArgs e)
         {
-            var errorMessage = e.Exception.ToString();
-            Debug.WriteLine(errorMessage);
-            await new MessageDialog(errorMessage, "An error occurred").ShowAsync();
+            Debug.WriteLine(e.Exception);
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/App.xaml.cs]: Why this change?

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Data/GenerateGeodatabase/GenerateGeodatabase.xaml.cs [hunk 4]
```diff
@@ -284,7 +284,7 @@ private void MapViewExtentChanged(object sender, EventArgs e)
             await Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, () =>
             {
                 // Update the progress bar value.
-                MyProgressBar.Value = _generateGdbJob.Progress / 1.0;
+                GenerateProgressBar.Value = _generateGdbJob.Progress / 1.0;
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Data/GenerateGeodatabase/GenerateGeodatabase.xaml.cs]: Why are we dividing by 1.0? 🤔

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Geometry/BufferList/BufferList.xaml.cs [hunk 5]
```diff
@@ -108,7 +108,7 @@ public BufferList()
             try
             {
                 // Get the boolean value whether to create a single unioned buffer (true) or independent buffer around each map point (false).
-                bool unionBufferBool = (UnionCheckBox.IsChecked == true);
+                bool unionBufferBool = UnionCheckBox.IsChecked == true;
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Geometry/BufferList/BufferList.xaml.cs]: Keep parens

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs [hunk 6]
```diff
@@ -139,7 +139,7 @@ private void GetSuitableTransformations(SpatialReference inSpatialRef, SpatialRe
             {
                 DatumTransformationListBoxItem item = new DatumTransformationListBoxItem(transform)
                 {
-                    IsDefault = (transform.Name == defaultTransform.Name)
+                    IsDefault = transform.Name == defaultTransform.Name
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs]: Keep parens

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml [hunk 7]
```diff
@@ -5,12 +5,16 @@
     xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
     <Grid>
         <esriUI:MapView x:Name="MyMapView" />
-        <Border Background="White" BorderBrush="Black" BorderThickness="1"
-                HorizontalAlignment="Right" VerticalAlignment="Top"
-                Margin="30" Padding="20" Width="375">
+        <Border Style="{StaticResource BorderStyle}">
+        <StackPanel>
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml]: Fix indentation

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml.cs [hunk 8]
```diff
@@ -8,21 +8,20 @@
...
 
 using Esri.ArcGISRuntime.Data;
-using Esri.ArcGISRuntime.Geometry;
 using Esri.ArcGISRuntime.Mapping;
 using Esri.ArcGISRuntime.Ogc;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml;
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml.cs]: FYI: UWP doesn't have same WebView-DPI-problem here that iOS did. So this WMS sample worked fine all along.

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Map/AuthorMap/AuthorMap.xaml.cs [hunk 9]
```diff
@@ -224,7 +218,7 @@ private void AddOperationalLayers()
             myMap.OperationalLayers.Clear();
 
             // Loop through the selected items in the operational layers list box
-            foreach (KeyValuePair<string,string> item in LayerListView.SelectedItems)
+            foreach (KeyValuePair<string,string> item in OperationalLayerListBox.SelectedItems)
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Map/AuthorMap/AuthorMap.xaml.cs]: Fix spacing between type params, while you're at it.

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Map/SearchPortalMaps/SearchPortalMaps.xaml.cs [hunk 10]
```diff
@@ -27,166 +28,138 @@ namespace ArcGISRuntime.UWP.Samples.SearchPortalMaps
...
         }
 
-        private async void ShowOAuthSettingsDialog()
+        private async void SearchButton_Click(object sender, RoutedEventArgs e)
         {
-            // Show default settings for client ID and redirect URL
-            ClientIdTextBox.Text = AppClientId;
-            RedirectUrlTextBox.Text = OAuthRedirectUrl;
+            // Get web map portal items in the current user's folder or from a keyword search
+            IEnumerable<PortalItem> mapItems = null;
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Map/SearchPortalMaps/SearchPortalMaps.xaml.cs]: Isn't assigning `null` redundant here?

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs [hunk 11]
```diff
@@ -142,7 +142,7 @@ public ListTransformations()
             {
                 DatumTransformationListBoxItem item = new DatumTransformationListBoxItem(transform)
                 {
-                    IsDefault = (transform.Name == defaultTransform.Name)
+                    IsDefault = transform.Name == defaultTransform.Name
```
Comment (fragmer) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs]: Keep parens

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/StyleWmsLayer/StyleWmsLayer.xaml [hunk 12]
```diff
@@ -35,6 +32,6 @@
                         IsEnabled="False"
                         Click="SecondStyleButton_Clicked" />
             </Grid>
-        </Border>
+            </Border>
```
Comment (fragmer) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/StyleWmsLayer/StyleWmsLayer.xaml]: Fix indentation

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Security/TokenSecuredChallenge/TokenSecuredChallenge.xaml [hunk 13]
```diff
@@ -7,106 +7,36 @@
...
-				HorizontalAlignment="Right" VerticalAlignment="Top">
+        <Border Style="{StaticResource BorderStyle}">
             <StackPanel>
                 <TextBlock FontSize="14" FontWeight="Bold"
-                           Margin="4" 
-                           Text="Map Layers (with status)" />
-                <ListView x:Name="listLayerStatus" 
-                          Padding="6" 
+                           Margin="4"
+                           Text="Map sayers (with status)" />
```
Comment (fragmer) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Security/TokenSecuredChallenge/TokenSecuredChallenge.xaml]: Fix typo in "layers"

File: src/iOS/Xamarin.iOS/Samples/Security/TokenSecuredChallenge/TokenSecuredChallenge.cs [hunk 14]
```diff
@@ -300,7 +300,7 @@ public LoginOverlay(CGRect frame, nfloat transparency, UIColor color, string url
             nfloat buttonWidth = 60;
 
             // Get the total height and width of the control set (five rows of controls, four sets of space).
-            nfloat totalHeight = (5 * controlHeight) + (4 * rowSpace);
+            nfloat totalHeight = 5 * controlHeight + 4 * rowSpace;
```
Comment (fragmer) [inline src/iOS/Xamarin.iOS/Samples/Security/TokenSecuredChallenge/TokenSecuredChallenge.cs]: Keep parens

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml.cs (hunk 8 above)
Comment (nCastle1) [reply src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml.cs]: I know, but when I updated the other samples to not use callouts you recommended that I do the same for desktop because showing arbitrary/unpredictable HTML in a callout is bad pattern to show.

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Hydrography/ChangeEncDisplaySettings/ChangeEncDisplaySettings.xaml.cs [hunk 15]
```diff
@@ -110,10 +111,17 @@ private void UpdateDisplaySettings()
...
         {
             // Apply display settings
-            UpdateDisplaySettings();
+            try
+            {
+                UpdateDisplaySettings();
+            }
+            catch (NullReferenceException)
+            {
+                // The sample wasn't ready
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Hydrography/ChangeEncDisplaySettings/ChangeEncDisplaySettings.xaml.cs]: How barbaric! I'm guessing that UWP attaches our checked event handler before setting IsChecked in XAML? I was hoping that there'd be a cleaner way to address this...

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/FeatureLayerRenderingModeMap/FeatureLayerRenderingModeMap.xaml.cs [hunk 16]
```diff
@@ -37,6 +37,10 @@ public FeatureLayerRenderingModeMap()
 
         private void Initialize()
         {
+            // Create the maps.
+            MyStaticMapView.Map = new Map();
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/FeatureLayerRenderingModeMap/FeatureLayerRenderingModeMap.xaml.cs]: How about setting InitialViewpoint here, like on other platforms? This UWP sample currently starts off zoomed waaay out.

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml [hunk 17]
```diff
@@ -5,12 +5,16 @@
...
-                Margin="30" Padding="20" Width="375">
-            <TextBlock Text="Tap to identify features."
-                       TextAlignment="Center"
-                       FontWeight="SemiBold" />
+        <Border Style="{StaticResource BorderStyle}">
+            <StackPanel>
+                <TextBlock Text="Tap to identify features."
+                           TextAlignment="Center"
+                           FontWeight="SemiBold" />
+                <WebView x:Name="ResultWebView"
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml]: We should probably update this sample's thumbnail image, which is still showing a callout.

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Map/SearchPortalMaps/SearchPortalMaps.xaml.cs [hunk 18]
```diff
@@ -15,6 +15,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Windows.UI.Popups;
+using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Map/SearchPortalMaps/SearchPortalMaps.xaml.cs]: `using Windows.UI.Xaml.Controls` is no longer needed.
